src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -123,12 +121,12 @@
       <PreprocessorDefinitions>WIN32;_DEBUG;_WINDOWS;_USRDLL;ASPNETCOREMODULE_EXPORTS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PrecompiledHeaderFile>stdafx.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)$(TargetName).pch</PrecompiledHeaderOutputFile>
-      <AdditionalIncludeDirectories>..\IISLib;.\Inc;..\CommonLib</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..\IISLib;.\Inc;..\CommonLib;;C:\Github\aspnetcore\.dotnet\packs\Microsoft.NETCore.App.Host.win-x64\7.0.0-alpha.1.21612.5\runtimes\win-x64\native</AdditionalIncludeDirectories>
```

Comment:
@dotnet/aspnet-build do you guys have any suggestions for what the proper way for me to reference into the packs\app.host would be here, I don't see anything that seems useful in the current msbuild variables.

Ideally I want something like `$(ReproRoot)\.dotnet\packs\Microsoft.NETCore.App.Host.win-$(Platform)\$(AppHostVersion)\runtimes\win-$(Platform)\native` so I can use it for x64/Win32/Arm64 without having to define this for each configuration

Reply:
`$(MicrosoftNETCoreAppRuntimeVersion)` is not specific to Microsoft.NETCore.App versions or platforms; that should do for `$(AppHostVersion)`. Do _not_ use `$(MicrosoftNETCoreBrowserDebugHostTransportVersion)` because that's a package version and doesn't provide information about local paths when on servicing branches; very useful for runtime, host, and related package downloads but not for here.

`$(Platform)` should be fine from what I'm seeing. Well, as long as `$(Platform)` gets the same values as in our WiX and C# projects (`arm`, `arm64`, `x64`, `x86`). I'm not positive other than the `win-XYZ` bit being consistent in the two parts of the path.

Reply:
Yeah I looked at AppRuntimeVersion already but that doesn't seem to matchup, here's what that results in today:

`LibNetHostPath = C:\Github\aspnetcore\\.dotnet\packs\Microsoft.NETCore.App.Host.win-x64\7.0.0-preview.2.22075.9\runtimes\win-x64\native`

But the version I get from restore seems to be: 22066.4

![image](https://user-images.githubusercontent.com/6537861/151478400-660e0be0-063d-4712-b8c5-2ac8e1c1dc7c.png)

Reply:
Oh, I was wrong about alignment. The .NET SDK provides a few options that match that version. See e.g. .dotnet\sdk\7.0.100-alpha.1.22071.3\Microsoft.NETCoreSdk.BundledVersions.props in your repo. `$(AppHostPackVersion)` has a fairly on-point name for this case and might work.

Reply:
Nice, so the last part is how do I reference that file, I see the sdk version in global.json but I don't see it hardcoded anywhere else as far as I can tell:

```
(aspnetcore) C:\Github\aspnetcore [haok/hostfxr ≡ +0 ~1 -0 !]> findstr /snip 22071.3 *.props
.dotnet\sdk\7.0.100-alpha.1.22071.3\Microsoft.NETCoreSdk.BundledVersions.props:24:    <NETCoreSdkVersion>7.0.100-alpha.1.22071.3</NETCoreSdkVersion>
```

Also I was just going to put this in the Directory.Build.Props for iis, but maybe we should put this in a shared props somewhere (maybe the RepoRoot props file?) since its probably useful 

```
  <Import Project="$(LocalDotNetRoot)\sdk\$(???)\Microsoft.NETCoreSdk.BundledVersions.props" />

```

Reply:
Shouldn't need to import the file at all. Any project using the .NET SDK should have the `$(AppHostPackVersion)` property defined.

Reply:
Are native projects somehow different then?  I wans't see any of the properties in the msbuild without explicitly importing

Reply:
Yes, very different. Native projects don't use the .NET SDK.

You may need to add the property to eng/tools/GenerateFiles/Directory.Build.props.in and change the `$(_TemplateProperties)` property in GenerateFiles.csproj to pass `$(AppHostPackVersion)`. Something like

``` msbuild
<AppHostPackVersion>${AppHostPackVersion}</AppHostPackVersion>
```

and

``` msbuild
AppHostPackVersion = $(AppHostPackVersion);
```

The final file is imported into most projects in the repo, including native projects.

Reply:
Oops, the bit for the template file should be conditional since we get it for free in most projects:

``` msbuild
<AppHostPackVersion Condition=" '$(AppHostPackVersion)' == '' ">${AppHostPackVersion}</AppHostPackVersion>
```

Reply:
Awesome, looks like this did the trick, whitespace apparently matters in those template files, i.e. `AppHostPackVersion=$(AppHostPackVersion)` otherwise it generates it as ' 7.0.0xxx' with a leading space